grails-app/assets/javascripts/meriplan.js

```diff
@@ -638,6 +650,8 @@ function DetailsViewModel(o, project, budgetHeaders, risks, config) {
     self.threats = new GenericViewModel(o.threats, ['threat', 'intervention']);
     self.consultation = ko.observable(o.consultation);
     self.communityEngagement = ko.observable(o.communityEngagement);
+    self.controlApproach = new GenericViewModel(o.controlApproach, ['data1', 'data2']);
```

Comment:
Rename "self.controlApproach" to "self.threatToNativeSpecies" as that is what is actually being recorded.

Reply:
I've pushed the changes for this